alphastats/plots/ClusterMap.py

```diff
@@ -58,7 +58,7 @@ def _prepare_df(self):
             significant_proteins = anova_df[anova_df["ANOVA_pvalue"] < 0.05][
                 Cols.INDEX
             ].to_list()
-            df = df[significant_proteins]  # TODO bug?
+            df = df[significant_proteins]  # TODO bug? df is not used again
```

Comment:
self.prepared_df = self.mat.loc[:, (self.mat != 0).any(axis=0)].transpose()
has to change to
self.prepared_df = df.transpose()
then all the code here actually matters